src/IndexService/ProductList/ElasticSearch/AbstractElasticSearch.php

```diff
@@ -321,19 +321,19 @@ public function getOrderKey(): array|string
     }
 
     /**
-     * Pass "unlimited" to do da Scroll Request
+     * Pass -1 to do da Scroll Request
```

Comment:
wanted to fix the typo 

```suggestion
     * Pass -1 to do the Scroll Request
```
but likely this would sound better
```suggestion
     * Pass -1 to enable the unlimited scroll request
```